src/command-line/utils.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+const colors = require("colors/safe");
+const fs = require("fs");
+const path = require("path");
+
+let loungeHome;
+
+class Utils {
+	static extraHelp() {
+		[
+			"",
+			"",
+			"  Environment variable:",
+			"",
+			`    LOUNGE_HOME   Path for all configuration files and folders. Defaults to ${colors.green(Utils.defaultLoungeHome())}.`,
+			"",
+		].forEach((e) => console.log(e));
```

Comment:
Yes, unfortunately, see their recommendation in https://github.com/tj/commander.js/issues/682 :(